README.md

```diff
@@ -155,84 +156,58 @@ def main():
     input_dicom_path = args.input
     output_dicom_path = args.output
 
-    extraAnonymizationRules = {}
+    extra_anonymization_rules = {}
 
-    def setupSeriesDescription(dataset, tag):
+    def setup_series_description(dataset, tag):
         element = dataset.get(tag)
         if element is not None:
-            element.value = element.value + '-' + args.suffix
+            element.value = f'{element.value}-{args.suffix}'
 
     # ALL_TAGS variable is defined on file dicomfields.py
     # the 'keep' method is already defined into the dicom-anonymizer
     # It will overrides the default behaviour
-    for i in allTags:
-        extraAnonymizationRules[i] = keep
+    for i in ALL_TAGS:
+        extra_anonymization_rules[i] = keep
 
     if args.suffix:
-        extraAnonymizationRules[(0x0008, 0x103E)] = setupSeriesDescription
+        extra_anonymization_rules[(0x0008, 0x103E)] = setup_series_description
 
     # Launch the anonymization
-    anonymize(input_dicom_path, output_dicom_path, extraAnonymizationRules)
+    anonymize(input_dicom_path, output_dicom_path, extra_anonymization_rules, delete_private_tags=False)
 
-if __name__ == "__main__":
+if __name__ == '__main__':
     main()
 ```
 
+See the full application in the `examples` folder.
+
 In your own file, you'll have to define:
 - Your custom functions. Be careful, your functions always have in inputs a dataset and a tag
 - A dictionary which map your functions to a tag
 
-## Anonymize dicom tags without dicom file
+## Anonymize dicom tags for a dataset
 
-If for some reason, you need to anonymize dicom fields without initial dicom file (extracted from a database for example). Here is how you can do it:
+You can also anonymize dicom fields in-place for pydicom's DataSet using `anonymize_dataset`. See this example:
 ```python
-from dicomanonymizer import *
+from dicomanonymizer import anonymize_dataset
+from pydicom.data import get_testdata_file
+from pydicom import dcmread
 
 def main():
-
-  # Create a list of tags object that should contains id, type and value
-  fields = [
-    { # Replaced by Anonymized
-      'id': (0x0040, 0xA123),
-      'type': 'LO',
-      'value': 'Annie de la Fontaine',
-    },
-    { # Replaced with empty value
-      'id': (0x0008, 0x0050),
-      'type': 'TM',
-      'value': 'bar',
-    },
-    { # Deleted
-      'id': (0x0018, 0x4000),
-      'type': 'VR',
-      'value': 'foo',
-    }
-  ]
-
-  # Create a readable dataset for pydicom
-  data = pydicom.Dataset()
-
-  # Add each field into the dataset
-  for field in fields:
-    data.add_new(field['id'], field['type'], field['value'])
-
-  anonymize_dataset(data)
```

Comment:
Why did you delete this example ? This can be useful.  
Why not moving it in the examples folder you created ?

Reply:
I tried this example and could not get it to work. I can fix the first warning about 'bar' not being valid for `TM`, but have no idea how to fix the error of missing `file_meta` for the dataset.

Here is the output:
```
C:\Users\smjoshi\.virtualenvs\examples-gxyY8XxM\Lib\site-packages\pydicom\valuerep.py:443: UserWarning: Invalid value for VR TM: 'bar'.
  warnings.warn(msg)
Traceback (most recent call last):
  File "C:\development\dicom-anonymizer\examples\anonymize_fields.py", line 33, in <module>
    main()
  File "C:\development\dicom-anonymizer\examples\anonymize_fields.py", line 30, in main
    anonymize_dataset(data)
  File "C:\development\dicom-anonymizer\dicomanonymizer\simpledicomanonymizer.py", line 436, in anonymize_dataset
    action(dataset.file_meta, tag)
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\smjoshi\.virtualenvs\examples-gxyY8XxM\Lib\site-packages\pydicom\dataset.py", line 908, in __getattr__
    return object.__getattribute__(self, name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Dataset' object has no attribute 'file_meta'
```

Reply:
That should be fixed with https://github.com/KitwareMedical/dicom-anonymizer/pull/61

Reply:
Thank you! I will pull in the changes into my branch and that should clean up this PR.